### PR TITLE
New feature: change user-agent for curl.

### PR DESCRIPTION
**Pull request by comsomisha**
Summary
Sometimes you need to change the user agent to save desktop pages in web.archive.org.
For example:
A) curl, default user-agent
https://web.archive.org/web/20200203111954/https://www.sports.ru/tribuna/blogs/puncher/2216091.html
Result: redirect to mobile version
B) curl, curl_user_agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.75 Safari/537.36"
https://web.archive.org/web/20200309090029/https://www.sports.ru/tribuna/blogs/puncher/2216091.html
Result: desktop version

# Changes these areas

- [ ] Bugfixes
- [x] Feature behavior
- [ ] Command line interface
- [ ] Configuration options
- [ ] Internal architecture
- [ ] Archived data layout on disk

### CODE REVIEW

**Comment by comsomisha** [comment]
P.S. I use option  '--restrict-file-names=nocontrol' because I save pages with Cyrillic links. Developers of ArchiveBox can save option '--restrict-file-names=windows'.

File: archivebox/archive_methods.py [hunk 1]
```diff
@@ -226,7 +227,7 @@ def fetch_wget(link_dir, link, timeout=TIMEOUT):
         '--span-hosts',
         '--no-parent',
         '-e', 'robots=off',
-        '--restrict-file-names=windows',
+        '--restrict-file-names=nocontrol',
```
Comment (pirate) [inline archivebox/archive_methods.py:230]: this doesn't work well on macOS unfortunately. can you make it a config option instead?

Comment (comsomisha) [reply archivebox/archive_methods.py:230]: I will create a config option "RESTRICT_FILE_NAMES".

**Comment by pirate** [comment]
thanks!

**Pull request merged**